room/runtime/src/main/java/androidx/room/SQLiteCopyOpenHelperFactory.java

```diff
@@ -32,24 +34,36 @@
     private final String mCopyFromAssetPath;
     @Nullable
     private final File mCopyFromFile;
+    private final boolean mIsCopyFromFileGzipped;
     @NonNull
     private final SupportSQLiteOpenHelper.Factory mDelegate;
 
     SQLiteCopyOpenHelperFactory(
             @Nullable String copyFromAssetPath,
             @Nullable File copyFromFile,
             @NonNull SupportSQLiteOpenHelper.Factory factory) {
+        this(copyFromAssetPath, copyFromFile, false, factory);
```

Comment:
kept this for backward compatibility, but since this is an internal class, can just remove it i guess

Reply:
(removed).